### ISSUE TITLE
desktopLauncher.closeSplashScreen() causing MacOSX app to hang

### ISSUE THREAD

**Issue by mofeejegi** [raeleus/skin-composer#35]
After trying the Mac jar, the normal jar and even the source code, the program never ran on my Mac. At best, it showed a blank screen that didn't agree to close. 

It was finally able to run though, after I commented out the above line (in the title). Apparently, your boolean flag only skips that line for Windows systems only. I have a hunch that it's the same reason why the jar file didn't run too.

**Comment by raeleus** [comment]
I've been trying to test this with no luck. I don't have access to a Mac. If you look at the Mac branch of Skin Composer, you'll see that [closeSplashScreen()](https://github.com/raeleus/skin-composer/blob/mac/desktop/src/com/ray3k/skincomposer/desktop/DesktopLauncher.java#L302) is an empty method. Commenting it out effectively does nothing.

Maybe you were using the main branch, explaining that part of it. With the Jar file, make sure you are extracting the jar from the zip. If you run it straight from the zip file, it crashes on all platforms. It needs write access to the directory and I should be checking for that with an appropriate error message if necessary.

**Comment by mofeejegi** [comment]
Wow! Then it's probably my fault. I simply downloaded the source without confirming which branch I was on. In my defense though, I just went straight to the downloads page from your site link. As for the jar files, I did the extraction properly. Actually, I didn't mention this earlier, the program did launch, and show the splash screen image, but crashed quickly after. That was solved using the run on first thread vmoption. The blank window came up when I used the vmoption, and that was it. In some particular sequences, my terminal window ended up hanging on my mac. Running it via Android Studio is definitely the way to go right now. 

Thanks for this awesome program.
